daml-lf/interface/src/main/scala/com/digitalasset/daml/lf/iface/Interface.scala

```diff
@@ -42,12 +42,22 @@ object InterfaceType {
   }
 }
 
+// Duplicate of the one in com.daml.lf.language to separate Ast and Iface
```

Comment:
If someone tells me that it’s fine to just expose parts of the `Ast`, I’m also perfectly happy with that. I don’t know what our policy here is.

Reply:
I think this is fine. Maybe we'll add a type parameter later :trollface:

Reply:
everything gets better if you add another type parameter.